bait-files/src/main/java/com/google/sps/servlets/RequestServlet.java

```diff
@@ -1,17 +1,29 @@
 package com.google.sps.servlets;
 
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
 import com.google.gson.Gson;
+import com.google.sps.servlets.Request;
 import com.googlecode.objectify.ObjectifyService;
 import java.io.IOException;
 import java.lang.String;
-import java.util.HashMap;
+import java.util.*;
```

Comment:
The Google style guide prefers importing all actual classes you need instead of using the wildcard, even though it takes more lines.

https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
This still looks like a wildcard